.github/workflows/deploy.yml

```
@@ -11,10 +11,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.9
+    - name: Set up Python
       uses: actions/setup-python@v2
```

OPTIONAL:
```suggestion
      uses: actions/setup-python@v4
```
https://github.com/actions/setup-python/releases

---

I think those updates are better suited for a different PR.